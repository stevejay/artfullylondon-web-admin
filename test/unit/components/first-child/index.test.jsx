import React from 'react'
import { shallow } from 'enzyme'

import FirstChild from '_admin/components/first-child'

it('should render only first child of array', () => {
  const wrapper = shallow(<FirstChild><div id='1' /><div id='2' /></FirstChild>)

  expect(wrapper).toMatchSnapshot()
})

it('should render only first child of single', () => {
  const wrapper = shallow(<FirstChild><div id='1' /></FirstChild>)
  expect(wrapper).toMatchSnapshot()
})

it('should handle no children', () => {
  const wrapper = shallow(<FirstChild />)
  expect(wrapper).toMatchSnapshot()
})
