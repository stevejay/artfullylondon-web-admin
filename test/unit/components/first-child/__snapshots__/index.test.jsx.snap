// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle no children 1`] = `""`;

exports[`should render only first child of array 1`] = `
<div
  id="1"
  key=".0"
/>
`;

exports[`should render only first child of single 1`] = `
<div
  id="1"
  key=".0"
/>
`;
